frontend/package.json

```diff
@@ -13,6 +13,7 @@
     "webdriver-update": "webdriver-manager update --versions.chrome=2.38",
     "test-gui-tap": "TAP=true yarn run test-gui",
     "test-gui-openshift": "yarn run test-suite --suite crud --params.openshift true",
+    "test-gui-servicecatalog": "yarn run test-suite --suite crud --params.servicecatalog true",
```

Comment:
This name makes it sound like you're running the service catalog suite, not the crud suite. I would just drop this shortcut.

We should consider turning openshift and servicecatalog on by default and leave the parameter to disable it (not necessary for this PR).